java/code/webapp/WEB-INF/nav/system_detail.xml

```diff
@@ -179,16 +179,27 @@
     <rhn-tab name="Deployment" url="/rhn/systems/details/virtualization/Deployment.do" acl="not system_is_virtual(); system_has_management_entitlement()"/>
   </rhn-tab>
 
-  <rhn-tab name="Audit" acl="system_has_management_entitlement() or system_has_salt_entitlement_and_contact_method(default)">
-    <rhn-tab-url>/rhn/systems/details/audit/ListScap.do</rhn-tab-url>
-    <rhn-tab name="nav.system.audit.list_scans">
+  <rhn-tab name="Audit" acl="system_has_management_entitlement() or system_has_salt_entitlement_and_contact_method(default)" url="/rhn/systems/details/audit/ListScap.do">
+    <rhn-tab name="OpenSCAP" url="/rhn/systems/details/audit/ListScap.do">
       <rhn-tab-url>/rhn/systems/details/audit/ListScap.do</rhn-tab-url>
-      <rhn-tab-url>/rhn/systems/details/audit/XccdfDetails.do</rhn-tab-url>
-      <rhn-tab-url>/rhn/systems/details/audit/RuleDetails.do</rhn-tab-url>
-      <rhn-tab-url>/rhn/systems/details/audit/XccdfDeleteConfirm.do</rhn-tab-url>
+      <rhn-tab name="nav.system.audit.list_scans">
+        <rhn-tab-url>/rhn/systems/details/audit/ListScap.do</rhn-tab-url>
+        <rhn-tab-url>/rhn/systems/details/audit/XccdfDetails.do</rhn-tab-url>
+        <rhn-tab-url>/rhn/systems/details/audit/RuleDetails.do</rhn-tab-url>
+        <rhn-tab-url>/rhn/systems/details/audit/XccdfDeleteConfirm.do</rhn-tab-url>
+      </rhn-tab>
+      <rhn-tab name="nav.system.audit.schedule">
+        <rhn-tab-url>/rhn/systems/details/audit/ScheduleXccdf.do</rhn-tab-url>
+      </rhn-tab>
```

Comment:
if this change the tab structure, we need to check the testsuite if we use this somewhere.

Reply:
Good point. I checked the testsuite and the tests for OpenSCAP do:
```
    When I follow "Audit" in the content area
    And I follow "Schedule" in the content area
    ...
```

This should work regardless of the change, since the `Schedule` link is still on the page that opens when you click on `Audit`. I will double check though.